booking_chart/mixin/mixin.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, *args, **kwargs):
         super(resource_mixin, self).__init__(*args, **kwargs)
     
     
-    def create(self, cr, uid, vals, context=None):
+    def create(self, cr, uid, vals, context=None, **kwargs):
```

Comment:
what is the purpose when you add extra `**kwargs` argument while openerp version 7 API does not have that ?